From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@LuisCabrer__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (luisca). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

MSSedusch committed:
Fix netmask

Commit:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master'

Commit:
doc updates 1

Commit:
Acrolinx, split long sentences, more bullets instead of long paragraphs, more detail about Demo-LoadGenerator.ps1. MightyPen = genemi.

mahesh-unnikrishnan committed:
New article: support for Azure CSP subscriptions.

YuriDio committed:
updating article

bwren committed:
change property

Commit:
Major rewrite attempt #1

Commit:
Freshness on manage-avail

Commit:
Rewrite attempt #2

Commit:
revision #3

Commit:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master'

mahesh-unnikrishnan committed:
Update active-directory-ds-csp.md

mahesh-unnikrishnan committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

huxuan committed:
Minor Fix for Face API doc.

davidbritch committed:
Removed Azure classic portal reference

The doc mentioned the Azure classic portal for retrieving a value from Active Directory. While it used to be the case that the classic portal had to be used to retrieve AD values, this is no longer the case.

mahesh-unnikrishnan committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

mahesh-unnikrishnan committed:
Updates to CSP article.

mahesh-unnikrishnan committed:
acrolinx fixes

mahesh-unnikrishnan committed:
Add to TOC.

Commit:
Free account updates

Commit:
fixed broken link

Commit:
fixed tooltip location

Commit:
Edits #3

jonbeck7 committed:
Update virtual-machines-common-sizes-memory.md

Commit:
freshness

Commit:
Fixing typo

Commit:
Supressed Cores

Commit:
Clear up, and erase, the two commented out ?? occurrences.

neilpeterson committed:
updated helm doc, updated toc

sethmanheim committed:
Rewrite to use .NET Standard client library

billmath committed:
updating portal references

sethmanheim committed:
Fixes

mahesh-unnikrishnan committed:
Update active-directory-ds-csp.md

spelluru committed:
Updates

Commit:
superscript!

sethmanheim committed:
Code fixes

Commit:
Fixing formatting issues

sethmanheim committed:
Adding image borders

spelluru committed:
no need of image

ktoliver committed:
Merge pull request #29549 from neilpeterson/helm-update

updated helm doc, updated toc

Commit:
Shortening a H2.

Commit:
Constrained vCPU

ktoliver committed:
Merge pull request #29552 from sethmanheim/sb-qs

Rewrite to use .NET Standard client library

rwike77 committed:
updating quickstarts

rwike77 committed:
minor updates

Commit:
Merge pull request #29540 from cynthn/custom-image-12-17

Custom Image Freshness

gbowerman committed:
Adding caveats about expired tokens, and adding smalldisk sku

spelluru committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into adfqs1207-2

Commit:
Updated the Scheduled Backup Limits

ktoliver committed:
Merge pull request #28706 from ChackDan/master

Master

ktoliver committed:
Merge pull request #29140 from MightyPen/saas-20171202-dpt-deploy

Acrolinx, and more.

spelluru committed:
H3 -> H4

Commit:
free account catherine's feedback

mahesh-unnikrishnan committed:
More updates to the article.

mahesh-unnikrishnan committed:
Merge branch 'master' of https://github.com/mahesh-unnikrishnan/azure-docs-pr

mahesh-unnikrishnan committed:
acrolinx fixes.

mahesh-unnikrishnan committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

MicrosoftGuyJFlo committed:
Remove Preview note

MicrosoftGuyJFlo committed:
Remove Preview note

PRMerger15 committed:
Merge pull request #29445 from cynthn/manage-avail-12-17

Freshness on manage-avail

Commit:
Merge pull request #29569 from gbowerman/patch-20

Adding caveats about expired tokens, and adding smalldisk sku

shengcMSFT committed:
Update transform-data-using-dotnet-custom-activity.md

adding V2 custom activity sample

huxuan committed:
File rename.

Commit:
toc

GitHubber17 committed:
Merge pull request #29560 from rwike77/lpupdate

updating quickstarts

GitHubber17 committed:
Merge pull request #29414 from mahesh-unnikrishnan/master

New article: support for Azure CSP subscriptions.

PRMerger10 committed:
Merge pull request #29555 from spelluru/adfqs1207-2

Updates

PRMerger10 committed:
Merge pull request #29572 from shengcMSFT/patch-15

Update transform-data-using-dotnet-custom-activity.md

GitHubber17 committed:
Merge pull request #29477 from huxuan/xuahu/FaceMinorFix

Minor Fix for Face API doc.

GitHubber17 committed:
Merge pull request #29561 from rwike77/tocupdate

minor updates

barclayn committed:
minor changes to reflect the current state of pen test rules

kanshiG committed:
Again fix the lazy index language

andredm7 committed:
Changes

ashishg2014 committed:
<azure to azure docs>

PRMerger17 committed:
Merge pull request #29576 from andredm7/andredm1207171

Changes

ashishg2014 committed:
<azure to azure docs1>

ashishg2014 committed:
<azure to azure docs1>

Commit:
Merge pull request #29479 from davidbritch/patch-1

Removed Azure classic portal reference

PRMerger16 committed:
Merge pull request #29585 from asgang/master

updated Azure to Azure documentation

MSSedusch committed:
Update sap-hana-high-availability.md

cephalin committed:
add cli backup scripts

cephalin committed:
acrolinx

MSSedusch committed:
Update high-availability-guide-suse.md

MSSedusch committed:
Update sap-hana-high-availability.md

MSSedusch committed:
Update sap-hana-high-availability.md

PRMerger13 committed:
Merge pull request #29587 from MSSedusch/patch-1

Update SAP HA on Linux articles

Commit:
Face Python Tutorial Typo Fix.

guyinacube committed:
Change Power BI Embedded to Workspace Collections

Commit:
Merge pull request #29590 from huxuan/xuahu/FacePythonTutorialTypoFix

Face Python Tutorial Typo Fix.

PRMerger14 committed:
Merge pull request #29591 from guyinacube/china

👊 Change Power BI Embedded to Workspace Collections

barclayn committed:
trying to fix note format

cephalin committed:
staged review

cephalin committed:
acrolinx

Commit:
Merge pull request #29420 from YuriDio/UpdatesNov_06

updating article

PRMerger18 committed:
Merge pull request #29592 from barclayn/pen-test

minor changes to reflect the current state of pen test rules

alkohli committed:
fixed an image for add a volume

PRMerger12 committed:
Merge pull request #29571 from MicrosoftGuyJFlo/MFAPreview

Remove Preview note

PRMerger14 committed:
Merge pull request #29553 from billmath/classicscrub

updating portal references

rjagiewich committed:
Merge pull request #29588 from cephalin/backup

add cli backup scripts

alkohli committed:
fine tuned the image

kanshiG committed:
Changed the language again

We need to discourage customers from using this feature - as many of them end up doing consistent index querying.

cwatson-cat committed:
Updating author so I can retire topic.

itechedit committed:
Merge pull request #29542 from cynthn/29537

Supressed Cores and superscript

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

PRMerger19 committed:
Merge pull request #29425 from bwren/la-arm

Log Analytics ARM - change property

PRMerger12 committed:
Merge pull request #29575 from kanshiG/patch-22

Again fix the lazy index language

PRMerger20 committed:
Merge pull request #29602 from cwatson-cat/12-8-DeleteOldCLILogin

Updating author so I can retire topic.

rjagiewich committed:
Merge pull request #29593 from alkohli/SSVol

fixed an image for add a volume

itechedit committed:
Merge pull request #29533 from amberbhargava/freeaccountchanges-11-9

Free account updates

jonbeck7 committed:
Update virtual-machines-common-sizes-general.md

v-alje committed:
Merge pull request #29603 from MicrosoftDocs/FromPublicRepo

From public repo

vladvino committed:
Update api-management-advanced-policies.md

rjagiewich committed:
Merge pull request #29565 from SyntaxC4-MSFT/app-service-limits

Updated Schedule Backup Limits for App Service

PatrickBue committed:
Update scenario-image-classification-using-cntk.md

Updating readme to support newest version of the workbench

PRMerger13 committed:
Merge pull request #29604 from jonbeck7/patch-14

Update virtual-machines-common-sizes-general.md

PRMerger12 committed:
Merge pull request #29605 from vladvino/patch-10

Update api-management-advanced-policies.md

PRMerger20 committed:
Merge pull request #29609 from PatrickBue/master

Update scenario-image-classification-using-cntk.md

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPrivateRepo
